app/validators/action_kit_fields.rb

```diff
@@ -67,6 +67,10 @@ class ActionKitFields < ActiveModel::Validator
     action_
   ).freeze
 
+  def initialize(name)
+    @name = name
```

Comment:
If you're worried `name` isn't a string, then perhaps call `to_s` on it here, rather than below? https://github.com/SumOfUs/Champaign/pull/853/files#diff-e52e089e43530d3d8f597e45e9128cfcR106